app/services/clients/adminusers_client.js

```diff
@@ -4,53 +4,62 @@
 const baseClient = require('./base_client/base_client')
 const requestLogger = require('../../utils/request_logger')
 const Service = require('../../models/Service.class')
-const { createCallbackToPromiseConverter } = require('../../utils/response_converter')
+const { getCounter } = require('../../metrics/graphite_reporter')
 
 // Constants
 const SERVICE_NAME = 'adminusers'
-
-const responseBodyToServiceTransformer = function responseBodyToServiceTransformer (body) {
-  try {
-    const service = new Service(body)
-    return Promise.resolve(service)
-  } catch (error) {
-    return Promise.reject(error)
-  }
-}
+const METRICS_PREFIX = 'internal-rest-call.adminusers'
+const SUCCESS_CODES = [200, 201, 202, 204, 206]
 
 let baseUrl
 let correlationId
 
-const findServiceBy = function findServiceBy (findOptions, loggingFields = {}) {
+/** @private */
+const _getAdminUsers = (url, description, findOptions, loggingFields = {}, callingFunction) => {
```

Comment:
There's no need for this method to be anonymous, this will help when looking at the stacktrace etc.
```suggestion
const _getAdminUsers = function _getAdminUsers(url, description, findOptions, loggingFields = {}, callingFunction) {
```